src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -723,7 +723,7 @@ export default class AuthorizationService {
 
   private static async processDynamicFilters(tenant: Tenant, userToken: UserToken, action: Action, entity: Entity,
       authorizationFilters: AuthorizationFilter, authorizationContext: AuthorizationContext, extraFilters?: Record<string, any>): Promise<void> {
-    if (!Utils.isEmptyArray(authorizationContext.filters)) {
+    if (userToken.role !== UserRole.ADMIN && !Utils.isEmptyArray(authorizationContext.filters)) {
```

Comment:
Could you please add a comment (TODO) here explaining a better way should be found to 'merge' the 'merge' the authorization if a user has several role? 

Thanks.